content/support/cybertec.md

```diff
@@ -0,0 +1,7 @@
+---
+title: CYBERTEC
+image: logo/CYBERTEC_LOGO_GREY_BLUE_SVG.svg
+homepage: https://www.cybertec-postgresql.com/
```

Comment:
Please ensure that the website mentions CloudNativePG and has a link back to cloudnative-pg.io. Thanks.